src/nidigital/examples/do_nothing_files/timing.digitiming

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
They are there because I needed something to test with to make sure everything worked as expected.

Once there are real nidigital examples, then they will be removed.

I was following the pattern with the support files in the system tests. Which may change in the future but as of now is per test.